File: python/35_parallelepiped.py
import math

a, b, c = map(int, input().split())
x = int(math.sqrt(a * c / b) + 0.5)
y = int(math.sqrt(a * b / c) + 0.5)
z = int(math.sqrt(b * c / a) + 0.5)
print(x, y, z)

File: c/35_parallelepiped.c
#include <stdio.h>
#include <math.h>
int main()
{
    long long a, b, c;
    if (scanf("%lld %lld %lld", &a, &b, &c) != 3)
        return 0;
    long long x = (long long)sqrt((double)a * c / b + 0.5);
    long long y = (long long)sqrt((double)a * b / c + 0.5);
    long long z = (long long)sqrt((double)b * c / a + 0.5);
    printf("%lld %lld %lld\n", x, y, z);
    return 0;
}
